cstar/marbl/component.py

```diff
@@ -5,13 +5,13 @@ class MARBLComponent(Component):
     # Inherits its docstring from Component
 
     def build(self) -> None:
-        print("source code modifications to MARBL are not yet supported")
+        pass
 
     def pre_run(self) -> None:
-        print("no pre-run actions involving MARBL are currently supported")
+        pass
 
     def run(self) -> None:
         print("MARBL must be run in the context of a parent model")
```

Comment:
Surely if this method is not intended to be called then?
```suggestion
        raise AttributeError("MARBL must be run in the context of a parent model")
```